### PR TITLE
Feature/pacstesting

### PR DESCRIPTION
**Pull request by jas88**
- Add unit test to reproduce issue seen in live PACS feed with retry causing errors with duplicate image fetches
- Add stub PACS to enable more comprehensive testing in future
- Change existing Orthanc test to use public PACS for C-ECHO testing instead of needing a separate PACS installation

### CODE REVIEW

File: Rdmp.Dicom.Tests/Unit/PacsFetch.cs [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                    else
+                    {
+                        pc.SetResult(DicomPresentationContextResult.RejectAbstractSyntaxNotSupported);
+                    }
+                }
+                return SendAssociationAcceptAsync(association);
+            }
+        }
+
+        private static readonly NopProgressListener Listener = new NopProgressListener();
```
Comment (tznind) [inline Rdmp.Dicom.Tests/Unit/PacsFetch.cs:88]: You can use `ThrowImmediatelyDataLoadEventListener` instead of having to have a private class.

File: Rdmp.Dicom.sln.DotSettings [hunk 2]
```diff
@@ -0,0 +1,4 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```
Comment (tznind) [inline Rdmp.Dicom.sln.DotSettings:1]: What is this file? Is it needed?

File: SharedAssemblyInfo.cs [hunk 3]
```diff
@@ -10,3 +11,4 @@
 [assembly: AssemblyVersion("2.1.7")]
 [assembly: AssemblyFileVersion("2.1.7")]
 [assembly: AssemblyInformationalVersion("2.1.7")]
+[assembly: InternalsVisibleTo("Rdmp.Dicom.Tests")]
```
Comment (tznind) [inline SharedAssemblyInfo.cs:14]: Is this still needed?

**Review by tznind — CHANGES_REQUESTED** [review]

**Review by tznind — APPROVED** [review]
